CZ/pyranda_rayleigh_taylor/rayleigh_taylor.py

```diff
@@ -248,21 +251,28 @@ def getPressure(pysim,p0):
     p0     = 1.0
     At     = ( (rho_h - rho_l)/(rho_h + rho_l) )
     # NOTE: WHY THIS PARTICULAR FRACTION OF THE RT GROWTH RATE?
-    u0     = sqrt( abs(gx*At/waveLength) ) * .1
+
 
     # Add interface
     # NOTE: diffuse interface?
-    :Yl: = .5 * (1.0-tanh( sqrt(pi)*( meshx - 1.4*pi ) / delta ))
+    :Yl: = .5 * (1.0-tanh( sqrt(pi)*( meshx ) / delta ))
     :Yh: = 1.0 - :Yl:
     :p:  += p0 
 
     # Add oscillations near interface
     wgt = 4*:Yh:*:Yl:
     :v: *= 0.0
     :w: *= 0.0
-    # NOTE: what is gbar?
+
+    # gbar is 2D gaussian filter (smoothes stuff out)
+    # Add random velocity (filtered white noise)
+    u0  = sqrt( abs(gx*At/waveLength) ) * ranVelMag
```

Comment:
do we really want this as a free parameter?   large values for kick starting things seems questionable?  aren't the modes the more significant factor in the late time behavior?

Reply:
This is somewhat equivalent to setting the amplitude of the perturbation; It should be bounded for sure but it could be useful to have available.